lib/mailer.js

```diff
@@ -284,7 +285,7 @@ const factory = (options) => {
     precompile: compile,
     render: render,
     send: sendEmail,
-    init
```

Comment:
Actually I am not renaming the public interface, because `Mailer.init` is still `Mailer.init.`

I am renaming the private interface to something different in order _not to _ overwrite this public interface with the extend call, because the way it was before the first call to `Mailer.init` overwrote the public `Mailer.init()` interface with the private one.

The goal of this was to enable several init calls to add templates and helpers from several places to the Mailer global. One in main app, and one in the package for example.

But I may have gone a bit too fast: it's not taking into account global helpers name overriding for example...